yt_dlp/extractor/vimeo.py

```diff
@@ -253,6 +253,23 @@ def _extract_original_format(self, url, video_id, unlisted_hash=None):
                             'quality': 1,
                         }
 
+    def _extract_original_format_2(self, video_id):
+        jwt_response = self._download_json('https://vimeo.com/_rv/viewer', video_id)
+        if jwt_response.get('jwt'):
+            headers = {'Authorization': f"jwt {jwt_response['jwt']}"}
```

Comment:
What's supposed to be the guidelines for quotes in an f string?
should it be `"` inside of `'` or other way round?

Reply:
I try not to use fstrings with nested quotes - they are hard to read and easy to make mistakes in. But if u must, I guess `'` in outer makes more sense with the rest of the code

Reply:
Neither was looking decent so i removed the f string.